pkg/util/validation/validate.go

```diff
@@ -34,16 +34,16 @@ const (
 	// ErrQueryTooLong is used in chunk store, querier and query frontend.
 	ErrQueryTooLong = "the query time range exceeds the limit (query length: %s, limit: %s)"
 
-	missingMetricName       = "missing_metric_name"
-	invalidMetricName       = "metric_name_invalid"
-	greaterThanMaxSampleAge = "greater_than_max_sample_age"
-	maxLabelNamesPerSeries  = "max_label_names_per_series"
-	tooFarInFuture          = "too_far_in_future"
-	invalidLabel            = "label_invalid"
-	labelNameTooLong        = "label_name_too_long"
-	duplicateLabelNames     = "duplicate_label_names"
-	labelsNotSorted         = "labels_not_sorted"
-	labelValueTooLong       = "label_value_too_long"
+	missingMetricName             = "missing_metric_name"
+	invalidMetricName             = "metric_name_invalid"
+	greaterThanMaxSampleAge       = "greater_than_max_sample_age"
+	maxLabelNamesPerSeries        = "max_label_names_per_series"
+	tooFarInFuture                = "too_far_in_future"
+	invalidLabel                  = "label_invalid"
+	labelNameTooLong              = "label_name_too_long"
+	duplicateLabelNames           = "duplicate_label_names"
+	labelsNotSorted               = "labels_not_sorted"
+	labelValueTooLong             = "label_value_too_long"
```

Comment:
Why did the formatting change here?

Reply:
Hum, there I can't really explain, I didnt change that myself so maybe it was a gofmt hook, but I see no changes for me, the old blame is kept:

![image](https://user-images.githubusercontent.com/2609731/135619493-b879b2f3-ac26-44bd-84ed-3346a8893b85.png)